usr/iscsi_util.c

```diff
@@ -392,3 +397,26 @@ int iscsi_match_target(void *data, struct session_info *info)
 				     info->persistent_port, NULL,
 				     MATCH_ANY_SID);
 }
+
+/*
+ * init thread's PR_SET_IO_FLUSHER flag
+ *
+ * val: 1 to set to set thread's PR_SET_IO_FLUSHER flag
+ *      0 to clear thread's PR_SET_IO_FLUSHER flag
+ *
+ * return: return 0 on success, else error number is returned
+ */
+int init_thread_io_flusher(int val)
```

Comment:
Maybe set_thread_io_flusher is a better name, because we are actually overriding the default that was set when the process was created and it was initialized.

Reply:
updated

Reply:
I believe he made this change, so resolving this conversation.